### PR TITLE
Skeletonize LED Subsystem

### PR DESCRIPTION
**Pull request by having11**
Adds the `LEDSubsystem` with a factory method for getting the associated color/pattern command based on the `RobotState`

### CODE REVIEW

File: src/main/cpp/moduledrivers/ConnectorX.cpp [hunk 1]
```diff
@@ -63,15 +64,15 @@ uint16_t ConnectorX::ConnectorXBoard::readAnalogPin(AnalogPort port) {
 void ConnectorX::ConnectorXBoard::setLedPort(LedPort port) {
   ConsoleLogger::getInstance().logInfo("requested led port", (int)port);
 
-  if (port != _currentLedPort) {
+  // if (port != _currentLedPort) {
```
Comment (nobody5050) [inline src/main/cpp/moduledrivers/ConnectorX.cpp:67]: Does this just disable port 0 entirely?

**Review by nobody5050 — APPROVED** [review]
LGTM

File: src/main/cpp/RobotContainer.cpp [hunk 2]
```diff
@@ -26,18 +26,18 @@
...
+#include <frc2/command/WaitCommand.h>
 
 using namespace DriveConstants;
 
 RobotContainer::RobotContainer() {
   // Initialize all of your commands and subsystems here
 
 #ifdef TEST_SWERVE_BOT
-  m_wrist = std::make_unique<WristSubsystem>();
+  // m_wrist = std::make_unique<WristSubsystem>();
```
[the remark this answers is not in the document]
Comment (having11) [reply src/main/cpp/RobotContainer.cpp:40]: thought that was out of scope of this PR

File: src/main/cpp/moduledrivers/ConnectorX.cpp (hunk 1 above)
Comment (having11) [reply src/main/cpp/moduledrivers/ConnectorX.cpp:67]: no, it only bypasses the cache check to always set the requested port

Comment (having11) [reply src/main/cpp/moduledrivers/ConnectorX.cpp:67]: The Rio is still so fast that the Pico is unable to keep up with two I2C transmissions in-a-row

**Pull request merged**